doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -287,9 +287,6 @@ libraries.
 | Object Storage: Swift | https://docs.openstack.org/api-ref/object-store/     | v1              |                              |
 | Placement             | https://docs.openstack.org/api-ref/placement/        | v1              | 1.36                         |
 | Orchestration: Heat   | https://docs.openstack.org/api-ref/orchestration/v1/ | v1              |                              |
-<!--
```

Comment:
Please do not delete

Reply:
Keep commented the content about Cyborg, as it may be reintroduced if we move to a higher OpenStack release